### ISSUE TITLE
helper-cli: Create command to import ORT scan result

### ISSUE THREAD

**Issue by tsteenbe** [oss-review-toolkit/ort#7404]
Proposing we implement an 'import-scan-results' ORT helper CLI command which imports the scan results into configured scan storage from given scan-results.json file.  Being able to import scan results saves a lot of compute / re-scanning. 

Example scenario the import-scan-results command will enable  
- Easier sharing scan results between parties for re-use or debugging purposes
- Import scan results from Eclipse foundation for internal organizational use.

**Comment by sschuberth** [comment]
Note that we already have [UploadResultToPostgresCommand](https://github.com/oss-review-toolkit/ort/blob/main/plugins/commands/upload-result-to-postgres/src/main/kotlin/UploadResultToPostgresCommand.kt). Doesn't that fulfill the purpose already?

**Comment by mnonnenmacher** [comment]
I agree with @fviernau's second proposal to have commands instead which can import/export results to/from a storage for the reasons given. If you need a filter this could be done by providing a list of identifiers or by providing an ORT result file to only export scan results relevant for the contained projects/packages.

**Comment by fviernau** [comment]
> I agree with @fviernau's second proposal to have commands instead which can import/export results to/from a storage for the reasons given. If you need a filter this could be done by providing a list of identifiers or by providing an ORT result file to only export scan results relevant for the contained projects/packages.

Nice idea, e.g. having separate import export commands. This makes it usable even if you do not have access to one of the storages. Like organization A makes an export optionally limiting it to a set of IDs. Hands over that export to organization B which makes the import, without having access to organization A's storage.
